src/back/auth/src/repo/table_state.hpp

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "../model/model.hpp"
+
+#include <string>
+#include <string_view>
+
+#include <userver/components/loggable_component_base.hpp>
+#include <userver/yaml_config/schema.hpp>
+#include <userver/utest/using_namespace_userver.hpp>
+#include <userver/storages/postgres/cluster.hpp>
+
+namespace svetit::auth::table {
+
+class State final {
+public:
+	static constexpr std::string_view kName = "state";
+
+	explicit State(
+	    storages::postgres::ClusterPtr pg);
```

Comment:
```suggestion
		storages::postgres::ClusterPtr pg);
```

Используем табы, а не пробелы.

Reply:
поменял везде.